### PR TITLE
https://issues.redhat.com/browse/ACM-7412 Removing the searchCustomization API for 2.9 (1/2)

### PR DESCRIPTION
**Pull request by dockerymick**
https://issues.redhat.com/browse/ACM-7412

### CODE REVIEW

File: apis/api.adoc [hunk 1]
```diff
@@ -17,7 +17,7 @@ For more information, review the API documentation for each of the following res
 * xref:../apis/helmreleases.json.adoc#helm-api[Helm API]
 * xref:../apis/policy.json.adoc#policy-api[Policy API]
 * xref:../apis/observability.json.adoc#observability-api[Observability API]
-* xref:../apis/search.json.adoc#search-api[Search customization API]
+//* xref:../apis/search.json.adoc#search-api[Search customization API]
```
Comment (dockerymick) [inline apis/api.adoc]:
```suggestion
//* xref:../apis/search.json.adoc#search-api[Search API]
```

File: apis/api.adoc [hunk 2]
```diff
@@ -17,7 +17,7 @@ For more information, review the API documentation for each of the following res
 * xref:../apis/helmreleases.json.adoc#helm-api[Helm API]
 * xref:../apis/policy.json.adoc#policy-api[Policy API]
 * xref:../apis/observability.json.adoc#observability-api[Observability API]
-* xref:../apis/search.json.adoc#search-api[Search customization API]
+//* xref:../apis/search.json.adoc#search-api[Search API]
```
Comment (dockerymick) [inline apis/api.adoc:21]: hid this link bc we will add the Search API

**Review by jlpadilla — APPROVED** [review]
/lgtm

**Review by cadawson — APPROVED** [review]
To clarify: We resolved the bug by removing the search customization API. Instead of removing it, you modified it refer to the search API. We will add the search API before GA.  If so, it looks good. Thanks!

**Comment by dockerymick** [comment]
@cadawson that is correct. I modified the link reference as a place holder and will add the Search API before GA. Thanks for your review and approval!

**Comment by dockerymick** [comment]
@jlpadilla thanks for your review and approval as well!

**Comment by jlpadilla** [comment]
@cadawson The `SearchCustomization` API was removed from the product in 2.7, so I want to cherry pick this to previous releases to avoid confusion.
We will also add back the new `Search` API.

File: apis/api.adoc (hunk 2 above)
Comment (dockerymick) [inline apis/api.adoc:21]:
```suggestion
//hid comment, this is place holder for Search API | MJ | 09/19/23
//* xref:../apis/search.json.adoc#search-api[Search API]
```

**Pull request merged**